packages/requisitions/src/ResponseRequisition/DetailView/AppBarButtons/SupplyRequestedQuantityButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  ButtonWithIcon,
+  PlusCircleIcon,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useSupplyRequestedQuantity } from '../../api';
+
+export const SupplyRequestedQuantityButtonComponent = () => {
+  const t = useTranslation('distribution');
+  const { mutate: supplyRequestedQuantity } = useSupplyRequestedQuantity();
+  const getConfirmation = useConfirmationModal({
+    onConfirm: supplyRequestedQuantity,
+    message: t('messages.supply-to-requested'),
+    title: t('heading.supply-to-requested'),
+  });
+
+  return (
+    <ButtonWithIcon
```

Comment:
I wonder if there's some way to disable the button if all lines have already been set to requested?
On my test requisition I've got a lot of lines with requested=0, which might not be possible? but does make things a bit strage.

We could just run though and check to see if all lines have requested=quantity to supply - there's nothing provided by the api is there?

Reply:
Nah, nothing by the API. Also Requested equal to zero I think is possible?

 I did wonder also if it was a good idea and this is the lazy way. 

I figured there was three options:
- Alert saying "Everything is already at the requested quantity"  and have the user push "OK"
- No alert, just send the request (my lazy way)
- No alert, no request.. just pretend it happened..

🤷 😁

Reply:
let's leave it - asking the user to press OK again to clear an alert is also annoying for some people.
will wait and see what other people think